tests/test_init.py

```diff
@@ -19,90 +19,241 @@
+                       (1,)),
+                      (qml.init.simplified_two_design_initial_layer_normal,
+                       {'n_wires': 3, 'mean': 0, 'std': 1},
+                       (3,)),
+                      (qml.init.simplified_two_design_weights_uniform,
+                       {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1},
+                       (0,)),
+                      (qml.init.simplified_two_design_weights_uniform,
+                       {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1},
+                       (2, 1, 2)),
+                      (qml.init.simplified_two_design_weights_uniform,
+                       {'n_layers': 2, 'n_wires': 4, 'low': 0, 'high': 1},
+                       (2, 3, 2)),
+                      (qml.init.simplified_two_design_weights_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (0, )),
+                      (qml.init.simplified_two_design_weights_normal,
+                       {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1},
+                       (2, 1, 2)),
+                      (qml.init.simplified_two_design_weights_normal,
+                       {'n_layers': 2, 'n_wires': 4, 'mean': 0, 'std': 1},
+                       (2, 3, 2)),
+                      (qml.init.basic_entangler_layers_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1)),
+                      (qml.init.basic_entangler_layers_normal,
+                       {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1},
                        (2, 2)),
-                      (qml.init.basic_entangler_layers_uniform, {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1},
+                      (qml.init.basic_entangler_layers_uniform,
+                       {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1},
                        (2, 1)),
-                      (qml.init.basic_entangler_layers_uniform, {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1},
+                      (qml.init.basic_entangler_layers_uniform,
+                       {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1},
                        (2, 2)),
                       ]
 # Functions returning a list of parameter arrays
-INITALL_KWARGS_SHAPES = [(qml.init.cvqnn_layers_all, rpt,
-                          [(repeat, n_if), (repeat, n_if), (repeat, n_wires), (repeat, n_wires), (repeat, n_wires),
-                           (repeat, n_if), (repeat, n_if), (repeat, n_wires), (repeat, n_wires), (repeat, n_wires),
-                           (repeat, n_wires)]),
-                         (qml.init.interferometer_all, base, [(n_if,), (n_if,), (n_wires,)])]
+INITALL_KWARGS_SHAPES = [(qml.init.cvqnn_layers_all, {'n_layers': 2, 'n_wires': 3},
+                          [(2, 3), (2, 3), (2, 3), (2, 3), (2, 3),
+                           (2, 3), (2, 3), (2, 3), (2, 3), (2, 3),
+                           (2, 3)]),
```

Comment:
It will be a reference to the same tuple 11 times, so let me know if that would cause a problem
```suggestion
                          [(2, 3)]*11),
```